### DISCUSSION TITLE
After updating to windows11 24h2, the screen tears when using rustdesk connection

### DISCUSSION THREAD

**Discussion by ColorfulDick** [rustdesk/rustdesk#10018] [General]
### Bug Description

as title，this issue is not present in other versions of Windows systems，same problem The same problem occurs in remote desktop in Windows system。how can i fix it？
![微信图片_20241122230209](https://github.com/user-attachments/assets/38b454fc-b456-4f91-a953-b95430ea16cf)

### How to Reproduce

use rustdesk(version1.2.3-1.3.2, installed in windows11 24h2) to control anothor computer

### Expected Behavior

Screen tearing

### Operating system(s) on local (controlling) side and remote (controlled) side

windows11 24h2

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.2.3-1.3.2

### Screenshots

![微信图片_20241122230209](https://github.com/user-attachments/assets/a1c720f8-5b69-482c-b60c-5695c72a4139)

**Comment by 21pages** [comment]
Thanks for your feedback, could you try different codec?

**Comment by ColorfulDick** [comment]
> Thanks for your feedback, could you try different codec?

i try to change the default encoding to h265/264，but this bug was not fixed; 

Are there any way can help me locate the problem?i want to know what cause this bug to appear

**Comment by ColorfulDick** [comment]
> Thanks for your feedback, could you try different codec?

same problem: https://answers.microsoft.com/zh-hans/windows/forum/all/%E6%9B%B4%E6%96%B0%E5%88%B0window11/fbdb4590-d5f6-4df7-ab76-197c12c4ab88

**Comment by 21pages** [comment]
Click the record button, if the record vedio is not good, it's a problem of the encoding. Try disable directx capture option in rustdesk general other setting on the controlled side. And try changing resolutions.

**Comment by rustdesk** [comment]
> > Thanks for your feedback, could you try different codec?
> 
> same problem: https://answers.microsoft.com/zh-hans/windows/forum/all/%E6%9B%B4%E6%96%B0%E5%88%B0window11/fbdb4590-d5f6-4df7-ab76-197c12c4ab88

So RDP has the same problem? If so, I do not think we are able to fix it. I will convert it to discussion.
